### ISSUE TITLE
monitor my watershed won't load, just spins

### ISSUE THREAD

**Issue by dbressler75** [ODM2/ODM2DataSharingPortal#477]
![error_monmw_wontload](https://user-images.githubusercontent.com/39309124/78047604-8e44cd00-7346-11ea-90c9-2ddd499d85da.PNG)

**Comment by dbressler75** [comment]
Problem seems to have resolved itself

**Comment by HeatherBrooks** [comment]
@dbressler75 @aufdenkampe As of 19:30 EDT, site is not loading. Reported on the [EnviroDIY forum](https://www.envirodiy.org/topic/monitormywatershed-org-is-not-responding/) by Matt Barney. I was able to replicate the issue. 
![image](https://user-images.githubusercontent.com/9368629/78084421-26ae7200-7386-11ea-896b-6b97a961e065.png)

**Comment by mbarneytu** [comment]
The site is not loading again, as of 8:54am MDT.

**Comment by HeatherBrooks** [comment]
@aufdenkampe @htaolimno @tslawecki Any insight on this sporadic site loading issue? Anthony and I were both able to reach the site shortly before @mbarneytu  commented that he couldn't. I checked when I saw Matt's comment and I couldn't connect; just a few minutes later I'm able to connect again.

**Comment by tslawecki** [comment]
Following the comment from @mbarneytu, I tried to connect to the web site and also did not succeed. I logged in to the server and saw that logs were reflecting continued activity throughout the apparent outage. Out of "an abundance of caution", I restarted the nginx, envirodiy, and tsa services. I noticed, however, that my earlier attempted connection had resolved successfully - possibly before the restart. I was also, of course, successful with a separate new website connection.

So, in summary, I don't yet know what to make of it - we have a self-resolving won't load, a won't load directly tied to a known internet outage, and this most recent one. I'll start by adding an Uptime Robot monitor, and also do a little diging into logs.

**Comment by aufdenkampe** [comment]
This is fixed at the moment, but we've definitely had a bunch of server-side issues in the last few days. Part of the issue was a 5.5 hour internet outage to our servers caused by a Comcast fiber-optic line issue on March 31 in the afternoon. But we’ve had other issues that we’re trying to track down.

If this happens to you again, please run a `traceroute` or `tracert` command from your computer and send us the results (maybe in an email to me). See https://en.wikipedia.org/wiki/Traceroute

I’m connecting this to the related discussion on the EnviroDIY forums at https://www.envirodiy.org/topic/response-code-504-from-data-envirodiy-org/

**Comment by dbressler75** [comment]
@aufdenkampe this issue has appeared again.  Seems to be the same with the entire monmw page not loading, "ERR_CONNECTION_TIMED_OUT".  Emailing tracert to Anthony

**Comment by tslawecki** [comment]
This is a server issue. LTI-IT is investigating.

**Comment by tslawecki** [comment]
The EnviroDIY VMs are now up and running. LTI-IT reports that the virtual host server rebooted unexpectedly around 1 AM, and did not restart the VMs. They are looking for the cause.
